test/fixtures/main/cleared-actual.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+import {defineMessages, FormattedMessage} from 'react-intl';
+
+const msgs = defineMessages({
+  content: {
```

Comment:
Test will be right, if you add unused message.
test/fixtures/main/cleared-expected.json should remain unchanged.